### PR TITLE
Configure script

### PR DESCRIPTION
**Pull request by cpollok**
Adds a script named config.py which lets users specifiy one or more top-level packages for which the repository should be configured. Meaning it places CATKIN_IGNORE files in every unneeded package, so only the necessary dependencies are built when calling catkin_make.

### CODE REVIEW

**Comment by gaya-** [comment]
@cpollok When you're done with the current task, could you please test out the script on a couple of different packages? E.g. cram_beginner_tutorial and the bullet tutorial? The package.xml-s should be quite clean now.

**Comment by gaya-** [comment]
@cpollok Please test this package with the current state of CRAM, fix any issues and document the script well, such that @Vanessa-rin can test it. Documentation should either go into README.md (which is not very good if the script sits in the root of the repo), in the code as commentaries or into the help prompt of the script. Write a comment here when you're ready, so @Vanessa-rin can start.
@Vanessa-rin, please come by my office when you're working on Thursday, so we can discuss this quickly. When you test @cpollok's script, you should write a little protocol of what you tested and what were the results. There is a Trello card in the "Test Pending" column for this PR, where the protocol should go.

**Comment by cpollok** [comment]
@gaya- I fixed an issue regarding the inter-dependencies in cram_3rdparty by just including the whole stack of packages by default. Now the script should work fine as long as the package.xmls are clean.
@Vanessa-rin I documented everything in the code, but just to use the script the help message should suffice. If you have any questions just ask me.

**Comment by Vanessa-rin** [comment]
Will test this Script in the next 2 days.

File: config.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    return parser.parse_args()
+
+if __name__ == '__main__':
+    args = parse_args()
+
+    crawl()
+    check_package_names()
+
+    if not check_for_high_level_packages(args.packages):
+        print(usage())
```
Comment (Vanessa-rin) [inline config.py]: This print(usage*()) gave me the **error** _python script NameError: name 'usage' is not defined_ after deleting the code line it worked flawless.

**Comment by gaya-** [comment]
@cpollok Could you please clean up this PR? Or actually, just copy your configure script into a new branch and make a new PR. I would like to have just one commit to merge with the script. Also, could you please find out where your `usage()` went that gave Vanessa a trouble. I could copy the file myself, but I thought it would be nice if your name was on the commit. Please do this as soon as possible. This week is a PR merging week.

**Pull request closed without merging**